index.html

```diff
@@ -201,5 +201,88 @@ <h3 class="white">Opieka nad grobami</h3>
         </p>
       </div>
     </section>
+    <section id="speaker">
+      <div class="section-row">
+        <div class="section-white">
+          <p class="text-black">
+            Współpracujemy z doświadczonym Mówcą Pogrzebowym, specjalizującym się w prowadzeniu <strong>modlitw w kaplicy pogrzebowej</strong> 
+oraz organizacji <strong>świeckich ceremonii pogrzebowych</strong>. 
+          </p>
+        </div>
+        <div class="section-white">
+          <div class="section-right-align">
+            <button>
+              <p class="button-text">MÓWCA POGRZEBOWY</p>
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+
+
+    <section id="contact">
+      <div id="contact-container">
+ 
+          <h3 class="contact-header">Kontakt</h3>
+
+
+       <div id="contact-two-columns"> 
+
+
+            <div id="contact-left">
+              <h4 class="contact">ZAKŁAD POGRZEBOWY<br/> Janina Cieśla</h4>
```

Comment:
It's always better to use regular text-transform for texts. If capitalization is needed, it's better to make it in CSS via `text-transform: uppercase` :)

Reply:
Corrected.